pkg/konfluxgen/konfluxgen.go

```diff
@@ -394,9 +394,6 @@ func makeValidName(n string) string {
 }
 
 func WriteFileReplacingNewerTaskImages(name string, data []byte, perm os.FileMode) error {
-	if !strings.HasSuffix(name, ".yaml") || !strings.HasSuffix(name, ".yml") {
-		return os.WriteFile(name, data, perm)
-	}
```

Comment:
This is the root case but it's also not necessary anymore as I initially was using a YAML-only approach but after I made it file type agnostic